### ISSUE TITLE
Don't include demo widgets in release archive

### ISSUE THREAD

**Issue by Vtec234** [EdAyers/ProofWidgets4#19]
Currently, `lake build` will build all `widget/*.tsx` files and place them under `build/js`. This is great for getting demos running easily, but we also use the same build target to create the `.tar.gz` archives that are distributed to packages that import ProofWidgets. This means that JS code built only for demos, including bundles libraries from NPM, ends up bloating the size of the archive. As of today it is not too bad - just 7M - but it could become a concern as we add more demos that bundle in large libraries. We should better have a separate `archive` target which produces the JS needed for `ProofWidgets.lean` and nothing more. It would be even better if Lake could do this for us.

**Comment by Vtec234** [comment]
This is no longer relevant after #165.
